pyproject.toml

```diff
@@ -32,14 +32,14 @@ pyarrow = ">=6.0.0"
 dask = { version = ">=2021.9.1", optional = true }
 pyspark = { version = "^3.2.0", optional = true }
 importlib-metadata = ">=4.8.1"
-click = ">=7.0,<8.1"
+click = ">=7.0"
 s3fs = ">=2022.1.0"
 tenacity = ">=5.0"
 kubernetes = { version = ">=12.0.0", optional = true }
 
 [tool.poetry.dev-dependencies]
 mypy = ">=0.910"
-black = ">=21.7b0"
+black = ">=21.7b0,<23.0"
```

Comment:
black only has a pre-release of version 23+, thus temporarily adding an upper bound.

Reply:
Removed since we have a formal release `23.1` for black.